### ISSUE TITLE
Slave JJazzLab to DAW

### ISSUE THREAD

**Issue by MusoCity** [jjazzboss/JJazzLab-X#90]
You could slave JJazzLab to a DAW with a simple vst that sends VstTimeInfo to JJazzLab.
Another way that's not as good, let it receive Song Position Pointer (SPP) from midi input.

**Comment by jjazzboss** [comment]
I agree that slaving JJazzLab to a DAW would be a great plus. The big problem is that the Java sequencer implementation does not support any slave mode (there is an API for it, but it's not implemented).
